.rubocop.yml

```diff
@@ -261,3 +261,6 @@ Style/IfInsideElse:
 # Sometimes we just want to 'collect'
 Style/CollectionMethods:
   Enabled: false
+
+Lint/UnneededDisable:
```

Comment:
This'll be able to be removed later once we upgrade to a RuboCop version that includes my Lint/MissingRequireStatement checker ... I'll submit a PR to their repo soon